static/js/auth.jsx

```diff
@@ -9,14 +9,6 @@ var self = module.exports = {
             if (cb) cb(true);
                 return
         }
-        self.getToken(username, pass, (res) => {
```

Comment:
Yes, previously were called two async call, log in and getToken. That means than login couldn't know about the token value and  become works only after second call(when token is already setted in localStorage, after previous call). Now token is returned with login status and we don't send one more ajax for token value.
In other case we should call first getToken, wait until it set value and only after this call login.